app/javascript/components/admin/site_settings/appearance/BrandColor.jsx

```diff
@@ -1,28 +1,29 @@
 import React from 'react';
 import { Stack } from 'react-bootstrap';
-import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSiteSettings';
 import BrandColorPopover from './BrandColorPopover';
+import useSiteSettingAsync from '../../../../hooks/queries/site_settings/useSiteSettingAsync';
 
 export default function BrandColor() {
-  const { data: siteSettings } = useSiteSettings();
+  const { data: brandColor } = useSiteSettingAsync('PrimaryColor');
+  const { data: brandColorLight } = useSiteSettingAsync('PrimaryColorLight');
```

Comment:
Why are we using useSiteSettingAsync instead of useSiteSettings?

Reply:
Atm there's no way to retrieve just 1 site setting without the siteSettingAsync - maybe something we should review at a later point